builtin/providers/aws/resource_aws_api_gateway_domain_name.go

```diff
@@ -52,6 +52,11 @@ func resourceAwsApiGatewayDomainName() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate_upload_date": &schema.Schema{
```

Comment:
Oh got it! sorry, first contributions here... new to Go & Terraform development... 😅